python:topology_unittest fails on darwin (Python 2.7.9)

$ bazel clean
$ bazel build --config=darwin heron/...
$ bazel test --config=darwin heron/...
Executed 87 out of 87 tests: 86 tests pass and 1 fails locally.
FAIL: //heron/common/tests/python:topology_unittest 

"""
============================= test session starts ==============================
platform darwin -- Python 2.7.9 -- py-1.4.27 -- pytest-2.6.4
collected 9 items

../../../../../../../../heron/common/tests/python/topology_unittest.py ...F.....

=================================== FAILURES ===================================
____________________ TopologyTest.test_set_execution_state _____________________

self = <topology_unittest.TopologyTest testMethod=test_set_execution_state>

```
def test_set_execution_state(self):
  # Set it to None
  self.topology.set_execution_state(None)
  self.assertIsNone(self.topology.execution_state)
```

> ```
> self.assertIsNone(self.topology.dc)
> ```
> 
> E     AttributeError: Topology instance has no attribute 'dc'

../../../../../../../../heron/common/tests/python/topology_unittest.py:28: AttributeError
====================== 1 failed, 8 passed in 0.52 seconds ======================
"""

--- a/heron/common/tests/python/mock_proto.py
+++ b/heron/common/tests/python/mock_proto.py
@@ -8,7 +8,7 @@ from heron.common.src.python import constants
 class MockProto:
   topology_name = "mock_topology_name"
   topology_id = "mock_topology_id"
-  dc = "mock_topology_dc"
+  cluster = "mock_topology_cluster"
   environ = "mock_topology_environ"
 
   def create_mock_spout(self,
@@ -148,7 +148,7 @@ class MockProto:
     estate = protoEState.ExecutionState()
     estate.topology_name = MockProto.topology_name
     estate.topology_id = MockProto.topology_id
-    estate.dc = MockProto.dc
+    estate.cluster = MockProto.cluster
     estate.environ = MockProto.environ
     return estate
 

--- a/heron/common/tests/python/topology_unittest.py
+++ b/heron/common/tests/python/topology_unittest.py
@@ -25,13 +25,13 @@ class TopologyTest(unittest.TestCase):
     # Set it to None
     self.topology.set_execution_state(None)
     self.assertIsNone(self.topology.execution_state)
-    self.assertIsNone(self.topology.dc)
+    self.assertIsNone(self.topology.cluster)
     self.assertIsNone(self.topology.environ)
 
     estate = MockProto().create_mock_execution_state()
     self.topology.set_execution_state(estate)
     self.assertEqual(estate, self.topology.execution_state)
-    self.assertEqual(MockProto.dc, self.topology.dc)
+    self.assertEqual(MockProto.cluster, self.topology.cluster)
     self.assertEqual(MockProto.environ, self.topology.environ)
 
   def test_set_tmaster(self):

